doc/Security.xml

```diff
@@ -4,12 +4,12 @@
   <info>
```

Comment:
It is hard to see what has really changed i.e. Added or removed as the document has been restructured.